homeassistant/components/here_weather/sensor.py

```diff
@@ -0,0 +1,99 @@
+"""Support for the HERE Destination Weather service."""
```

Comment:
This should be a weather platform ?

Reply:
I was told to remove the weather platform: https://github.com/home-assistant/core/pull/28910#discussion_r471122342

Reply:
`HERE Destination Weather` is the name of the API

Reply:
Ha, oops. Yeah, I think it should be the other way around. The weather entity is the important one, the sensors not so much.

Reply:
Added back the weather platform